operator/apis/lifecycle/v1alpha3/keptntaskdefinition_webhook_test.go

```diff
@@ -0,0 +1,67 @@
+package v1alpha3
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+func TestKeptnTaskDefinition_ValidateFields(t *testing.T) {
```

Comment:
let's also add some tests to cover the `Validate[Create/Update/Delete]()` methods

Reply:
Alright I will do that